RecoMuon/MuonIdentification/python/muons1stStep_cfi.py

```diff
@@ -20,7 +20,7 @@
 
     fillEnergy = cms.bool(True),
     # OR
-    maxAbsPullX = cms.double(4.0),
+    maxAbsPullX = cms.double(3.0),
```

Comment:
Are you worried about 1GeV or less muons in B=0T case getting multiple scattering underestimated? It's non-issue in my opinion. 4sigma matching cut was very loose when it was introduced before Run1. All TrackerMuon selectors use at least 3 sigma cut or tighter. 4-sigma gives too many fakes in real data.